packages/api-server/src/methods/modules/gw.ts

```diff
@@ -202,7 +202,7 @@ export class Gw {
       args[0] = formatHexNumber(args[0]);
       args[1] = formatHexNumber(args[1]);
 
-      const result = await this.readonlyRpc.gw_get_nonce(...args);
+      const result = await this.rpc.gw_get_nonce(...args);
```

Comment:
Should also update https://github.com/nervosnetwork/godwoken-web3/blob/main/packages/godwoken/src/client.ts#L85